eos-upgrade-eos2-to-eos3

```diff
@@ -306,6 +306,15 @@ refspec=eos:os/eos/$prod/eos3
 EOF
 
 if ! $SKIP; then
+    # Add updated keys for verifying OS updates, as the one included in OS 2.6
+    # expired in 2019. For a period, OS 3.6 updates were signed with the
+    # (later-expiring) Flatpak key which pre-dates OS 2.6, so we also include it.
+    for key in \
+      https://raw.githubusercontent.com/endlessm/eos-keyring/master/keys/signing/EOSK1.pub \
+      https://raw.githubusercontent.com/endlessm/eos-keyring/master/keys/signing/EFSK1.pub; do
+      wget -nv -O - ${key} | ostree remote gpg-import --stdin eos
```

Comment:
I'm pretty sure from reading the code that the answer is yes, but I hope that this adds to the keyring rather than replacing its contents.

Reply:
Yes, I intentionally tested on my system with just EOSK1 and it was still able to verify a 3.6 commit that was only signed with EFSK1. So this per-repo keyring is additive to the platform trusted keys.